openapi/src/main/java/io/micronaut/openapi/view/RedocConfig.java

```diff
@@ -31,13 +31,14 @@
  */
 final class RedocConfig extends AbstractViewConfig {
 
+    public static final String REDOC_PREFIX = "redoc.";
     private static final String DEFAULT_REDOC_JS_PATH = OpenApiViewConfig.RESOURCE_DIR + "/";
 
     private static final List<String> RESOURCE_FILES = Collections.singletonList(
         DEFAULT_REDOC_JS_PATH + "redoc.standalone.js"
     );
 
-    private static final Map<String, Object> DEFAULT_OPTIONS = Collections.emptyMap();
+    private static final Map<String, Object> DEFAULT_OPTIONS = new HashMap<>();
```

Comment:
should be wrapped in immutable or better use `Collections.singletonMap`

Reply:
fixed